cocos/bindings/auto/jsb_pipeline_auto.h

```diff
@@ -9,13 +9,10 @@
 #include "cocos/renderer/pipeline/forward/ForwardStage.h"
 #include "cocos/renderer/pipeline/shadow/ShadowFlow.h"
 #include "cocos/renderer/pipeline/shadow/ShadowStage.h"
-#include "cocos/renderer/pipeline/ui/UIFlow.h"
-#include "cocos/renderer/pipeline/ui/UIStage.h"
```

Comment:
CI works now, do not have to change it manullay.

Reply:
> CI works now, do not have to change it manullay.

Fixed!